andela/travel_tool_front
# #160196032 Approve/Reject Request (By Requester's Manager)

#### What does this PR do?
- add a check to render approval buttons on My Approvals page only if requests are open
- show updated requests status on request details modal
- disable buttons when updating or when a request has been updated before

#### Description of Task to be completed?
add method to 
-  render ` Approve && Reject ` buttons only on ` My Approvals ` page
-  render buttons based on [mockup](https://projects.invisionapp.com/d/main#/console/14861108/314016836/preview) if a requests status has been updated
- disable both buttons if a requests status has been updated or isUpdating

#### How should this be manually tested?
- Clone the repo - https://github.com/andela/travel_tool_front.git
- Switch to the repo directory - cd travel_tool_front
- Install dependencies and start the app - make start
- Create a new request on My Requests tab and ensure you're listed as your manager
- Log in, click Requests tab on the left sidebar and select My Approvals
- Click an open request
- Any button out of Approve and Reject can be clicked to update the status of the request
- Click an open request on `My Requests && My Approvals ` page to display modal
- Click on an approved or rejected request and click both buttons

#### Any background context you want to provide?
- Create a request on ` My Requests ` page and don't forget to add yourself as manager on the form
- This [PR](https://github.com/andela/travel_tool_back/pull/39/files) on the backend is required for this to work as expected.

#### What are the relevant pivotal tracker stories?
[#160196032](https://www.pivotaltracker.com/story/show/160196032)

#### Screenshots (if appropriate)
<img width="1434" alt="screen shot 2018-09-09 at 11 20 11 pm" src="https://user-images.githubusercontent.com/23608060/45269470-f55e1080-b486-11e8-8da3-ad69de27e815.png">


#### Questions:
